### PR TITLE
Add comments and vision

### PR DESCRIPTION
**Pull request by razhakin**

### CODE REVIEW

File: Robot.java [hunk 1]
```diff
@@ -73,13 +65,13 @@ public void robotInit() {
       camera.setExposureAuto();
     }).start();
     bosch.counterInit();
```
Comment (Statej) [inline Robot.java]: #17

File: Robot.java [hunk 2]
```diff
@@ -99,7 +91,7 @@ private void configureTalon(WPI_TalonSRX talon) {
 		talon.configContinuousCurrentLimit(30, Constants.timeOutMs); // Must be 5 amps or more
```
Comment (Statej) [inline Robot.java:90]: #17

**Review by Statej — CHANGES_REQUESTED** [review]

File: Robot.java (hunk 1 above)
Comment (razhakin) [reply Robot.java]: Working on version with limit switches, will be addded later

File: Robot.java (hunk 2 above)
Comment (razhakin) [reply Robot.java:90]:
```suggestion
		talon.configContinuousCurrentLimit(40, Constants.timeOutMs); // Must be 5 amps or more
```
Agreed, changing to 40

File: Robot.java [hunk 3]
```diff
@@ -99,7 +91,7 @@ private void configureTalon(WPI_TalonSRX talon) {
...
 		talon.configPeakCurrentLimit(30, Constants.timeOutMs); // 100 A
```
Comment (Statej) [inline Robot.java:91]: change to 45

**Review by Statej — CHANGES_REQUESTED** [review]

File: Robot.java (hunk 3 above)
Comment (razhakin) [reply Robot.java:91]:
```suggestion
		talon.configContinuousCurrentLimit(40, Constants.timeOutMs); // Must be 5 amps or more
		talon.configPeakCurrentLimit(40, Constants.timeOutMs); // 100 A
```

Changed

**Review by Statej — APPROVED** [review]

**Review by KaplanL — APPROVED** [review]

**Pull request merged**